app/config/findbugs.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+
+<!--
+FindBugs exclude filter
+Severity: HARD
+-->
+<FindBugsFilter>
```

Comment:
Build fails when it doesn't fine some id in the data binding. Which means build checks data binding files too.